### ISSUE TITLE
Cohort pathways increased clarity in labeling in execution page

### ISSUE THREAD

**Issue by jhardin29** [OHDSI/Atlas#1139]
### Request
Increased clarity in path details:
1. When clicking on a portion of the ring label displays %Remain and %Diff; can these be changed to be more clear; something like "% in Path" and "% not in path" or maybe % on Drug and % not on drug? Can you also include the Persons with count and the denominator to make it very clear? 
2. When clicking on portion in second outer ring where there is a pathway with more than 1 drug or event - it is unclear to me (not sure of others) what the %remain means - is this the percent that were on drug1 who were also on drug2; so is the % out of the total persons with pathway count or the % out of the total persons with pathway from drug 1? It could be useful to include the numerator and denominator which would make it clear. 
Increased clarity in overall analysis details:
3. In the Utilities similar to the cohort generation tab could there be an export tab that lists in text view the analysis description?

**Comment by AntonStepanof** [comment]
For items 1 and 2:
Currently for Path details there are columns:
	remain - percent from pathway count
	diff - difference with previous row
I guess options "Not in path/Not on drug" don't describe that properly. 
Looks like adding of Persons count can be a good idea for increasing clarity.
It can be these options:
"Name", "% in Path", "% Diff with previous", "Persons count"   
For item 3:
I think we should discuss content for text view.

**Comment by pbr6cornell** [comment]
I agree additional documentation to describe exactly what the statistics represent is important to add as part of ATLAS 2.8.

**Comment by pbr6cornell** [comment]
@chrisknoll  just wrote documentation for Cohort Pathway, so this may be done very soon.
